docker/docker-compose.common.yml

```diff
@@ -0,0 +1,45 @@
+version: "3"
+
+services:
+  proxy:
+    image: dockercloud/haproxy
+    environment:
+      - BALANCE=leastconn
+      - EXTRA_GLOBAL_SETTINGS=debug
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    ports:
+      - 80:80
+    networks:
+      - openpension
+
+  database:
+    image: postgres
+    volumes:
+      - ./database:/etc/cron.daily
+      - /var/backups:/var/backups
```

Comment:
This mount our [database](https://github.com/hasadna/open_pension/tree/master/database) directory to the `etc/cron.daily` e.g. put the `autopgsqlbackup` script in `cron.daily` so the corn will run it daily. This is the backup script. It create the backups in `var/backups` so I mount it as well (to save the backups on the server and not in the container).

We don't need to mount those two volumes in `dev` env.